### DISCUSSION TITLE
Wondering what Notifications method do you all use?

### DISCUSSION THREAD

**Discussion by Neogx** [djdd87/SynoAI#119] [General]
I tired using home assistant before but couldn't get that to work so gave up but I'm wondering what you all use so i can try

**Comment by mkammes** [comment]
I use Telegram and it works very well. Lag is a few seconds from the detection.

I've had nothing but issues trying to set it up in Home Assistant, and using HA for notifications.  I've scrapped trying to make it work, but I'd prefer to have it integrated into HA. Then again, my HA going down is more likely than Telegram going down, so perhaps HA isn't the most reliable solution.

**Comment by djdd87** [comment]
I have personally found that I get the best results with telegram. HA should be working fine though.

**Comment by Neogx** [comment]
Ill give telegram a go then thanks!
